upi/openstack/03_bootstrap.yaml

```diff
@@ -25,15 +25,15 @@
     command:
       cmd: "openstack port set --tag {{ cluster_id_tag }} {{ os_port_bootstrap }}"
 
-  - name: 'Create the second bootstrap server port'
+  - name: 'Create the ACI containers bootstrap server port'
     os_port:
-      name: "{{ os_port_bootstrap2 }}"
-      network: "{{ os_network2 }}"
+      name: "{{ os_aci_containers_port_bootstrap }}"
+      network: "{{ os_aci_containers_network }}"
```

Comment:
os_opflex_network

Reply:
this one is fine (i was using opflex terminology before but i changed it)